mica/starcheck/starcheck.py

```diff
@@ -37,6 +39,74 @@ def get_timeline_at_date(date, timelines_db=None):
         % (date, date))
 
 
+def get_catalog_at_date(date, config=None, timelines_db=None):
```

Comment:
I'd really prefer to have this out of something like cmd_states or dynamic cmd_states, but ...